package.json

```diff
@@ -20,10 +20,10 @@
     "db:studio": "prisma studio",
     "setup": "run-s db:generate db:push db:seed",
     "start": "cross-env NODE_ENV=production node ./build/server.js",
-    "start:e2e": "cross-env NODE_ENV=testing node --require dotenv/config ./build/server.js",
+    "start:e2e": " node --require dotenv/config ./build/server.js",
```

Comment:
Why this change?

Reply:
I suggested this change ([slack conversation here](https://seneca-open-source.slack.com/archives/C04HR4WV5P0/p1677286047709169)) to get the e2e tests to work as intended. 

To summarize, I found that `testing` is not an acceptable option for `NODE_ENV`, but `test` is.
Also, the `pretest:e2e:run` script that does the build will replace all instance of `process.env.NODE_ENV` with its actual value. So, we needed to pass it there instead because it assumed production otherwise. Though there is an extra empty space on this line

Reply:
Thanks for this, sounds good.  Yes, let's fix the blank line.  Question: will the `NODE_ENV` carry forward to this script from the `pre*` script, or should we also add it here?

Reply:
I am not sure if it matters, since the `pre*` script will replace all instances of `process.env.NODE_ENV` in the code with `test`.  From my testing, it most likely won't carry forward